apps/acrs/translations/src/en/buttons.json

```diff
@@ -10,5 +10,6 @@
     "confirm-submission": "Submit",
     "continue-only": "Continue",
     "enter-form": "Enter form",
-    "exit": "Exit"
+    "exit": "Exit",
+    "sign-in-again": "Sign in again"
```

Comment:
If you want to change the button text for the HOF template you'll probably need to say `"start-again": "Sign in again"` here. The view code from HOF that generates this HTML looks like this:

```
{{<error}}
  {{$content}}
    <h1 class="govuk-heading-l">{{{content.title}}}</h1>
    <h2 class="govuk-heading-m">{{{content.message}}}</h2>
    <a href="{{startLink}}" class="govuk-button" role="button">{{#t}}buttons.start-again{{/t}}</a>
  {{/content}}
{{/error}}
```

I haven't tried this with the button text update, but worth a go. You would need to add this button to the 'verify' app list of buttons too.

Reply:
Done